app/services/check_breaks_in_work_history.rb

```diff
@@ -0,0 +1,36 @@
+class CheckBreaksInWorkHistory
```

Comment:
There is lots of logic in this class that IMHO better suites to living in a`WorkHistory` (class or something similarly named) which would operate over the collection of work experiences. Then the service could be very thin (or disappear completely).

Reply:
Given where we are in the delivery, feel free to ignore.